backend/src/test/java/hanglog/trip/presentation/CommunityControllerTest.java

```diff
@@ -0,0 +1,167 @@
+                                        .attributes(field("constraint", "yyyy-MM-dd")),
+                                fieldWithPath("trips[].endDate")
+                                        .type(JsonFieldType.STRING)
+                                        .description("여행 종료 날짜")
+                                        .attributes(field("constraint", "yyyy-MM-dd")),
+                                fieldWithPath("trips[].description")
+                                        .type(JsonFieldType.STRING)
+                                        .description("여행 요약")
+                                        .attributes(field("constraint", "200자 이하의 문자열")),
+                                fieldWithPath("trips[].likeCount")
+                                        .type(JsonFieldType.NUMBER)
+                                        .description("좋아요 숫자")
+                                        .attributes(field("constraint", "0과 양의 정수")),
+                                fieldWithPath("trips[].isLike")
+                                        .type(JsonFieldType.BOOLEAN)
+                                        .description("좋아요 유무")
+                                        .attributes(field("constraint", "boolean (true : 좋아요)")),
+                                fieldWithPath("trips[].imageUrl")
+                                        .type(JsonFieldType.STRING)
+                                        .description("여행 대표 이미지")
+                                        .attributes(field("constraint", "이미지 URL")),
+                                fieldWithPath("trips[].cities")
+                                        .type(JsonFieldType.ARRAY)
+                                        .description("여행 도시 배열")
+                                        .attributes(field("constraint", "1개 이상의 도시 정보")),
+                                fieldWithPath("trips[].cities[].id")
+                                        .type(JsonFieldType.NUMBER)
+                                        .description("여행 도시 ID")
+                                        .attributes(field("constraint", "양의 정수")),
+                                fieldWithPath("trips[].cities[].name")
+                                        .type(JsonFieldType.STRING)
+                                        .description("여행 도시 이름")
+                                        .attributes(field("constraint", "도시")),
+                                fieldWithPath("lastPageIndex")
+                                        .type(JsonFieldType.NUMBER)
+                                        .description("마지막 페이지 인덱스")
+                                        .attributes(field("constraint", "양의 정수"))
+                        )
+                ))
+                .andReturn();
+
+        final CommunityTripsResponse communityTripsResponses = objectMapper.readValue(
+                mvcResult.getResponse().getContentAsString(),
+                new TypeReference<>() {
+                }
```

Comment:
```suggestion
                CommunityTripsResponse.class
```

이렇게 하면 안되는 건가요?
다른 컨트롤러 테스트와 달라서요!

Reply:
오 저는 TripControllerTest에 있는 getTrips()의 테스트를 따왔던건데... 개인적으론 CommunityTripsResponse.class를 선호하니 디노의 의견을 체택하겠슴다

Reply:
[cbf2455](https://github.com/woowacourse-teams/2023-hang-log/pull/587/commits/cbf2455b4eff9fb3e879c002b406b53ea3945a15)